qiskit/circuit/library/generalized_gates/gr.py

```diff
@@ -0,0 +1,234 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""
+Global R gates.
+"""
+
+import numpy as np
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+from qiskit.circuit.library.standard_gates import RGate, RZGate
+
+
+class GR(QuantumCircuit):
+    r"""Global R gate.
+
```

Comment:
Can you add a sentence explaining that "global" means that it acts on all/some qubits on the system and maybe one about masking qubits such that it doesn't act on all? I think this would be very helpful 🙂

Reply:
Global gates act on all qubits and could be used in a transpiling pass to reduce the total number total gates. Masking/shelving qubits from the global gate could be a feature of future architectures and implemented here.